src/components/UserLabel/UserLabel.scss

```diff
@@ -0,0 +1,138 @@
+    border-radius: 20px;
+
+    transition-property: background-color;
+    transition-duration: $transitionDuration;
+    transition-timing-function: $transitionTimingFunction;
+
+    &_view_outlined {
+        &:after {
+            position: absolute;
+            z-index: -1;
+            inset: 0;
+            content: '';
+            border: 1px solid var(--g-color-line-generic);
+            border-radius: 20px;
+
+            transition-property: border-color;
+            transition-duration: $transitionDuration;
+            transition-timing-function: $transitionTimingFunction;
+        }
+    }
+
+    &_empty {
+        &#{$block}_size_s {
+            padding-inline-start: 12px;
+        }
+
+        &#{$block}_size_m {
+            padding-inline-start: 16px;
+        }
+    }
+
+    &_clickable:hover {
+        cursor: pointer;
+        background-color: var(--g-color-base-simple-hover);
+
+        &:after {
+            border-color: transparent;
+        }
+    }
+
+    &__main {
+        @include mixins.button-reset();
+
+        display: inline-flex;
+        align-items: center;
+        border-radius: inherit;
+        padding-inline-end: 6px;
+
+        #{$block}_closeable & {
+            padding-inline-end: 0;
+        }
+
+        #{$block}_clickable & {
+            outline-offset: -1px;
+
+            &:focus {
+                outline: 2px solid var(--g-color-line-focus);
+            }
+
+            &:focus:not(:focus-visible) {
+                outline: 0;
+            }
+        }
+    }
+
+    &__avatar {
+        --g-avatar-background-color: var(--g-color-base-generic-accent);
+        --g-avatar-color: var(--g-color-text-primary);
+
+        display: flex;
+
+        #{$block}_size_m & {
+            margin-inline-end: 12px;
+        }
+
+        #{$block}_size_s & {
+            margin-inline-end: 6px;
+        }
+    }
+
+    &__text {
+        #{$block}_size_m & {
+            @include mixins.text-body-2;
+            margin-inline-end: 10px;
+        }
+
+        #{$block}_size_s & {
+            @include mixins.text-body-1;
+            margin-inline-end: 6px;
+        }
+    }
+
+    &__close {
+        @include mixins.button-reset();
+
+        box-sizing: initial;
+        display: inline-flex;
+        justify-content: center;
+        align-items: center;
+        width: 16px;
+        cursor: pointer;
+        padding-inline-end: 6px;
+        color: var(--g-color-text-secondary);
+
+        transition-property: color;
+        transition-duration: $transitionDuration;
+        transition-timing-function: $transitionTimingFunction;
+
+        &:hover {
+            color: var(--g-color-text-primary);
+        }
+    }
+
+    &__close-icon {
+        border-radius: var(--g-focus-border-radius);
+
+        #{$block}__close:focus & {
+            outline: 2px solid var(--g-color-line-focus);
+        }
+
+        #{$block}__close:focus:not(:focus-visible) & {
+            outline: 0;
+        }
```

Comment:
Same as above, and why it's on the icon, not button itself?

Reply:
I fixed `:focus-visible`. We show a border for the icon because the bounding box for the button is too big and It looks worse.